docs/fatjar-regressions-v0.35.1-SNAPSHOT.md

```diff
@@ -0,0 +1,251 @@
+# Anserini Fatjar Regresions (v0.35.1-SNAPSHOT)
+
+## MS MARCO V1 Passage
+
+Currently, Anserini provides support for the following models:
+
++ BM25
```

Comment:
IIRC, there are many more choices in Pyserini 2cr. Are they Pyserini only? or there is no regression test for them?

Reply:
Yes, Pyserini has more conditions - many of which are not available from Anserini. I haven't though through relationship moving forward... i.e., are they going to have the same (unlikely); is Pyserini going to be a superset (more likely). My current thinking is that Anserini is going to have regressions that are "easily doable" from Lucene.